Assets/Character/Scripts/Movement.cs

```diff
@@ -3,13 +3,21 @@
 
 public class Movement : MonoBehaviour {
```

Comment:
Change the name of this script to be more specific to what it does exactly. There will be many types of movement in this game so the name of this script needs to be more specific.